hadoop-hdds/config/src/main/java/org/apache/hadoop/hdds/conf/ConfigurationSource.java

```diff
@@ -37,9 +39,6 @@
 
   char[] getPassword(String key) throws IOException;
 
-  @Deprecated
```

Comment:
> One possible approach is to introduce two interfaces: `ConfigSource` (read-only) and `ConfigDestination` (write only). `OzoneConfiguration` can implement both.

Thanks @elek for the suggestion.  I was considering this, but needed this nudge to implement it.

Reply:
>  I was considering this, but needed this nudge to implement it.

Oh, I see why. It makes the patch bigger because we use this dirty hack to store the port in the config.